yjit/src/codegen.rs

```diff
@@ -6347,8 +6347,9 @@ fn jit_thread_s_current(
     true
 }
 
-// Check if we know how to codegen for a particular cfunc method
-fn lookup_cfunc_codegen(def: *const rb_method_definition_t) -> Option<MethodGenFn> {
```

Comment:
What is wrong with calling it cfunc? It's hooked into only `gen_send_cfunc` at the moment, so we probably shouldn't imply it handles ISEQs as well.

Reply:
Yes, I agree lookup_cfunc_codegen() is a better name currently.